### ISSUE TITLE
BUG: Latest update 2.0.1 "apple_news_is_paid" breaks editor

### ISSUE THREAD

**Issue by rhayncf** [alleyinteractive/apple-news#666]
![image](https://user-images.githubusercontent.com/32179197/64347512-1259f080-cfc2-11e9-85ec-2b0653f5ded2.png)
TypeError: Cannot read property 'apple_news_is_paid' of undefined
    at /wp-content/plugins/publish-to-apple-news/build/pluginSidebar.js?ver=2.0.1:1:40651
    at n (/wp-includes/js/dist/data.min.js?ver=4.4.0:1:15092)
    at new r (/wp-includes/js/dist/data.min.js?ver=4.4.0:1:15359)
    at Yf (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:67:42)
    at qg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:92:302)
    at hi (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:104:285)
    at Qg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
    at Z (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:156:492)

**Comment by kujain** [comment]
I am seeing the same issue with a similar error log:
```
TypeError: Cannot read property 'apple_news_is_paid' of undefined
    at /wp-content/plugins/publish-to-apple-news/build/pluginSidebar.js?ver=2.0.1:1:40651
    at n (/wp-includes/js/dist/data.min.js?ver=4.4.0:1:15092)
    at new r (/wp-includes/js/dist/data.min.js?ver=4.4.0:1:15359)
    at Yf (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:67:42)
    at qg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:92:302)
    at hi (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:104:285)
    at Qg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
    at Kc (/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:155:16)
```

This happens for every post than gets published to AN. Please help!

**Comment by kujain** [comment]
@rhayncf I was able to get around this error by downloading the plugin package from [Github releases](https://github.com/alleyinteractive/apple-news/releases/tag/v2.0.1). I would have assumed that this version of the official wordpress.org version are the same, but it appears not to be so in some critical area. 

But after doing so, I lose the option to show a publish meta box on post editor - oh well, but at least I can continue working on my tests while we can dig deeper.

**Comment by kevinfodness** [comment]
In version 2, the publish metabox is replaced by a Gutenberg sidebar with the Apple News icon, similar to Jetpack. We didn't encounter that error in our tests, and the version in GitHub should match the version from WordPress.org, with the exception of a few development files not being included on the WordPress.org version. Additionally, I tested the WordPress.org version after releasing it and didn't see this issue, so it's possible that there is something else going on that's affecting the plugin's ability to load that value. We will take a look!

**Comment by kujain** [comment]
Thanks @kevinfodness ! To check further, I activated a default Twenty Nineteen theme and checked the post editor. Still no sidebar for Apple News (on posts) but it is visible on the classic editor UI. I checked the Options list for the menu UI and didn't see Apple News as a checkable option here as well:
![image](https://user-images.githubusercontent.com/120631/64624091-1b7b0100-d3b8-11e9-817e-b20fd42a60a9.png)

**Comment by jomurgel** [comment]
@kujain with Gutenberg enabled on a post, can you confirm that this icon/sidebar group does, in fact, not exist?

![Screen Shot 2019-09-16 at 9 53 54 AM](https://user-images.githubusercontent.com/5230729/64973280-f8dd6200-d867-11e9-9d4b-b277ca1d8a8e.png)

**Comment by kujain** [comment]
@jomurgel Thanks for the screencap. I can confirm that the News icon doesn't show up in the block editor, even though enabled for the default post type. To test this, I installed a vanilla 2019 theme with no other plugin enabled.
![image](https://user-images.githubusercontent.com/120631/64977197-f0ddec00-d880-11e9-878e-6fc5d88e564c.png)
![image](https://user-images.githubusercontent.com/120631/64977210-f6d3cd00-d880-11e9-981a-89fb2bc8de22.png)

**Comment by jomurgel** [comment]
@kujain thanks for confirming. 

Looking into this we may have a potential fix in an upcoming 2.0.2 release.

**Comment by kevinfodness** [comment]
This should be fixed with version 2.0.2 which just shipped today.
